src/yunohost/app.py

```diff
@@ -140,7 +144,7 @@ def app_fetchlist(url=None, name=None):
             raise MoulinetteError(errno.EBADR,
                                   m18n.n('appslist_retrieve_error', error=str(e)))
 
-        if (applist_request.status_code != 200):
+        if applist_request.status_code != 200:
```

Comment:
Yeah, that sound better, and also change the return code (if we know how to do that...).